index.js

```diff
@@ -12,7 +13,7 @@ function Chalk(options) {
 }
 
 // use bright blue on Windows as the normal blue color is illegible
-if (process.platform === 'win32') {
+if (process.platform === 'win32' && !xtermRegexp.test(process.env.TERM || 'dumb')) {
```

Comment:
Fully agree that there's no need to declare the regex. This is not a hot code path (very much not so)